templates/cornellLink/template.html

```diff
@@ -0,0 +1,15 @@
+{{template "header"}}
+Hey there {{.User.Name | fname}},<br />
+<br />
+You're getting this email because you just linked your MyHomeworkSpace account to Cornell! Congrats! A few things to keep in mind:
+<br />
+<ul>
+	<li>Some classes that you're automatically enrolled in by your college, such as ENGRG 1050, may not appear in your schedule. We're working on fixing this issue, but in the meantime, we reccomend manually adding these classes as calendar events.</li>
+	<li>If you make changes to your schedule (such as add, drop, or swap a class or section), you will need to re-import your schedule to MyHomeworkSpace. You can do this in settings, by disconnecting Cornell University and then following the instructions to add it again.</li>
+	<li>If you run into any issues, we're here to help! You can reach out to us to get help, request an email, or just say hi using the "Feedback" button in MyHomeworkSpace, or emailing <a href="mailto:hello@myhomework.space">hello@myhomework.space</a>!</li>
+</ul>
+<br />
+Thanks,
+<br />
+The MyHomeworkSpace Team
+{{template "footer"}}
```

Comment:
I really don't think an email is the best way to communicate this, especially since this stuff applies to the other schools too. It's definitely something we don't handle well, but I don't think an email is the best way to do it. See MyHomeworkSpace/client#75

(also, if you know there are some classes missing, then fix it! or is it more complicated than that?)

Reply:
It's more complicated than that. ENGRG 1050 is handled differently than other classes because all first-year engineering students are automatically enrolled in it, so it doesn't use the general enrollment system. I need to figure out what other classes are like that that I'm not taking. I know A&S has an A&S Seminar, but I don't think it works the same way, and _maybe_ CALS does, but IDK how that works either.

Reply:
Hmmm...maybe that disclaimer about those classes should be during the enrollment then? Like after it's finished, you have an intermediate stage on the login that says that the those classes aren't there. Although even then it feels a little sketchy (as in people might not notice the missing class), so it would be great if there were some sort of way to fix that...is there no central schedule system or something?

Reply:
I might be able to get Student Center to give it to me, I'll take a look :smile:

Reply:
Ok I was able to fix it! I'm removing the confirmation email now